packages/react-instantsearch/src/core/createInstantSearchManager.js

```diff
@@ -29,6 +29,9 @@ export default function createInstantSearchManager({
   helper.on('result', handleSearchSuccess);
   helper.on('error', handleSearchError);
 
+  let derivedHelpers = {};
```

Comment:
If I am mistaken because of some of the flow then let me know and we comment it maybe